teddy.js

```diff
@@ -387,6 +370,39 @@
 
       renderedTemplate = teddy.templates[template] || renderedTemplate
 
+      // check if noparse or noteddy tag exist
+      noparse = renderedTemplate.match(/\snoparse/g)
+      noteddy = renderedTemplate.match(/\snoteddy/g)
```

Comment:
Two things to consider here:
1. ` noparse` and ` noteddy` could be matched in a single regex
2. This match needs to be more defensive as `renderedTemplate` at this stage is the entire template. I think the best idea would be to create a regex that looks specifically for a string that starts with `<include`... and ends with ` noparse>` or ` noteddy>`

Reply:
noparse/noteddy may not always be the last attribute

it should also match this: `<include noteddy src='whatever'>` and similar